src/hott/06-contractible.rzk.md

```diff
@@ -296,13 +297,12 @@ A type equivalent to a contractible type is contractible.
 
 For example, we prove that based path spaces are contractible.
 
-```rzk
--- Transport in the space of paths starting at a is concatenation.
+```rzk title="Transport in the space of paths starting at $a$ is concatenation"
```

Comment:
Same as above:

```suggestion
```rzk title="Transport in the space of paths starting at a is concatenation"
```